applications/web_app/frontend/react_with_rest/template/nginx.conf

```diff
@@ -1,6 +1,6 @@
 server {
 
-  listen $PORT;
```

Comment:
But the problem here is that in Heroku you can not assume port 80. You need to use the port Heroku assigns for you..

Reply:
I checked what was running in heroku and it worked with port 80